### PR TITLE
1.3.4.3 Update (API 1.26)

### PR DESCRIPTION
**Pull request by Zaicon**

### CODE REVIEW

**Comment by WhiteXZ** [comment]
Update the submodule and provide a changelog please

**Comment by ProfessorXZ** [comment]
Missing an API tick (TSAPI 1.26)

**Comment by Enerdy** [comment]
tfw this PR has 1.26 in its title and yet there is no API tick.

File: CHANGELOG.md [hunk 1]
```diff
@@ -22,6 +22,12 @@ This is the rolling changelog for TShock for Terraria. Use past tense when addin
 * Added hardened stone to the whitelist of tiles editable by players (@DogooFalchion).
 * Added conversion system to send convert old MOTD format into smart text, while preserving initial line starting values to keep byte optimization for background colors Thanks to (@WhiteXZ, @Simon311, and especially @DogooFalchion) for the hard work on this issue.
 
+## TShock 4.3.21
+* Compatibility with Terraria 1.3.3.4 (@Zaicon)
+* API: Version tick 1.26
+* API: Depreciated PlayerDamage and PlayerKillMe packets (now uses PlayerHurtV2 and PlayerDeathV2)
```
Comment (WhiteXZ) [inline CHANGELOG.md]: Reword that to deprecated

File: CHANGELOG.md [hunk 2]
```diff
@@ -22,6 +22,12 @@ This is the rolling changelog for TShock for Terraria. Use past tense when addin
...
+* API: Main.rand now uses UnifiedRandom instead of Random
```
Comment (WhiteXZ) [inline CHANGELOG.md]: Maybe add a note: This **WILL** break any plugin that uses `Main.rand`

File: CHANGELOG.md [hunk 3]
```diff
@@ -22,6 +22,12 @@ This is the rolling changelog for TShock for Terraria. Use past tense when addin
 * Added hardened stone to the whitelist of tiles editable by players (@DogooFalchion).
 * Added conversion system to send convert old MOTD format into smart text, while preserving initial line starting values to keep byte optimization for background colors Thanks to (@WhiteXZ, @Simon311, and especially @DogooFalchion) for the hard work on this issue.
 
+## TShock 4.3.21
```
Comment (WhiteXZ) [inline CHANGELOG.md]: Everything in Upcoming Changes should be included in this

**Pull request merged**